Add article about Pyvo @ EP

clytaemnestra opened the pull request:
I’ve put together a draft of the article. Please feel free to add photos, notes, or anything else you have

hroncok commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+
+Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
+
+Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně ...
```
> kdo upstream udržuje 200 knihoven

Tohle mi úplně nedává smysl/nesedí. Cc @befeleme

clytaemnestra replied:
já jsem to takto pochopila z popisu přednášky, ale pokud máš návrh, jak to lépe říct, tak ho klidně přidej přes _Add a suggestion_ a akceptuju ho

hroncok replied:
Nechám to na @befeleme.
V práci knihovny udržuje v downstreamu a je jich cca 4 tisíce.

clytaemnestra replied:
aktualizováno, díky @befeleme!

hroncok commented:
Do the photos need some attribution?

clytaemnestra commented:
> Do the photos need some attribution?

They're official EP photos from here: https://www.flickr.com/photos/europython/albums. I'm not sure how it works for the blog.

clytaemnestra commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+
+Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
+
+Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně ...
```
```suggestion
Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!).
```

hroncok commented:
> > Do the photos need some attribution?
> 
> They're official EP photos from here: https://www.flickr.com/photos/europython/albums. I'm not sure how it works for the blog.

<img width="1080" height="1726" alt="Screenshot_20250904_232048_Firefox" src="https://github.com/user-attachments/assets/7214473c-ac65-4dc1-bda3-93668e8fedad" />

honzajavorek commented:
@moisesguimaraes might have an idea?

befeleme commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+
+Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
+
+Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně ...
```
```suggestion
Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolina měla přednášku o tom, jak by si přála, aby vypadaly Python projekty z pohledu balíčkování, metadat a testovatelnosti. Jakožto vývojařka Fedora Linuxu každý rok integruje cca 4000 Python knihoven do distribuce, což jí dává poměrně široký záběr na svět tzv. upstreamu. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!).
```

clytaemnestra commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+Shodou okolností vyšla konference právě na třetí středu v měsíci, tedy na den, kdy se v Praze pravidelně koná Pyvo. Tentokrát se tak lokální tradice spojila s EuroPythonem a výsledkem byl velký společný pokec u piva. Dorazilo kolem 200 lidí, což z něj dělá jedno z největších Pyv vůbec, stejně jako v roce 2023 na stejném místě. 
+
+Pyvo proběhlo v hospodě Na Hradbách, tradičně oblíbeném místě účastníctva EuroPythonu, kousek od Kongresového centra. Přednášky nebyly, jen zahrádka, hospodská atmosféra a spousta rozhovorů.
+
```
```suggestion

Na Pyvu se odehrála zajímavá událost. Framework Django letos slaví 20 let od svého vzniku, a tak organizace Django Software Foundation přinesla dort, o který se podělila s účastníky. Pyvo se dokonce objevilo na [oficiálním Django blogu](https://www.djangoproject.com/weblog/2025/sep/05/dsf-at-europython-2025-celebrating-20-years/).
```

clytaemnestra replied:
chceme toto zmínit?

clytaemnestra commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+
+Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolina měla přednášku o tom, jak by si přála, aby vypadaly Python projekty z pohledu balíčkování, metadat a testovatelnosti. Jakožto vývojařka Fedora Linuxu každý rok integruje cca 4000 Python knihoven do distribuce, což jí dává poměrně široký záběr na svět tzv. upstreamu. Lumí ...
+
+![lidé přednášející]({static}/images/ep2025.png)
```
```suggestion
![lidé přednášející]({static}/images/ep2025.png)
*Převzato z [EuroPython Flickr účtu](https://www.flickr.com/photos/europython/albums) s povolením organizátorů*
```

clytaemnestra commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,21 @@
+
+A při závěrečné session se ukázalo, kolik hudebníků se v komunitě sešlo: společně zahráli *Always Look on the Bright Side of Life*.
+
+![lidé hrající na nástroje]({static}/images/ep20251.png)
```
```suggestion
![lidé hrající na nástroje]({static}/images/ep20251.png)
*Převzato z [EuroPython Flickr účtu](https://www.flickr.com/photos/europython/albums) s povolením organizátorů*
```

clytaemnestra commented:
I've changed the license in Flickr to CC BY-NC-SA. That means that for non-commercial use, we can use photos with attribution. I've added the attribution.

Can you please do a final review, @hroncok? If you're happy with the current state, please approve or suggest changes. Thanks!

befeleme commented on content/2025-09-04_pyvo-na-ep.md:
```diff
@@ -0,0 +1,24 @@
+Gravatar: miabajic.miabajic@gmail.com
+About: Mia je vývojářka, místopředsedkyně EuroPython Society, a spoluorganizátorka pražského Pyva. 
+
+Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
```
```suggestion
Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnáct set účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
```

befeleme commented:
Jeden nitpick, jinak za me OK. Diky!

befeleme commented:
@hroncok, would you like to take another look?

hroncok commented:
Nepotřebuju, díky.
